src/app/content/search/hooks/index.ts

```diff
@@ -97,13 +97,12 @@ export const receiveSearchHook: ActionHookBody<typeof receiveSearchResults> = (s
 };
 
 export const clearSearchHook: ActionHookBody<typeof clearSearch | typeof openToc> = (services) => () => {
-  const scrollTarget = selectNavigation.scrollTarget(services.getState());
-  if (scrollTarget && isSearchScrollTarget(scrollTarget)) {
-    services.history.replace({
-      hash: '',
-      search: '',
-    });
-  }
```

Comment:
Want to confirm that it's ok to remove the below test, which is now failing. I don't understand why the search string would not be cleared without a scrollTarget.
`hooks > clearSearchHook > noops if there was no scroll target or if it wasn't search scroll target`